tests/regionizers/test_h3_regionizer.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for H3Regionizer."""
+
+import osmnx as ox
+import pytest
+
+from srai.regionizers.h3_regionizer import H3Regionizer
+
+
+@pytest.mark.parametrize(  # type: ignore
+    "geoname,resolution,buffer",
+    [
+        ("Wroclaw, Poland", 7, True),
+        ("Tokyo, Japan", 7, False),
+        ("London, England", 7, True),
+    ],
+)
+def test_transform(geoname: str, resolution: int, buffer: bool) -> None:
```

Comment:
Maybe some more tests? Check for example what happens when resolution out of range is passed, and what happens if empty GDF is passed (like in the voronoi regionizer tests).

Reply:
Yep, need to add some more tests